### ISSUE TITLE
Updates / dev tool improvements

### ISSUE THREAD

**Issue by tsa96** [momentum-mod/website#1079]
[This check](https://github.com/momentum-mod/website/blob/6b9d227dda974359dfbf98ad7eafb67883e6b34d/apps/backend-e2e/src/admin.e2e-spec.ts#L1938) is failing inconsistently. My instinct is an un-awaited Promise or similar, but can't tell where.

**Comment by tsa96** [comment]
Will try look at this at some point but others welcome to take a look.

**Comment by GordiNoki** [comment]
Yep. I just wasn't sure if my fix for sure fixed the problem, but looks like it did
